src/app/modules/gb-export-module/gb-export.component.html

```diff
@@ -1,134 +1,149 @@
 <div class="gb-export-container">
-  <!---------------------- data table ---------------------->
-  <div class="gb-export-section-banner">
-    <h5>View and arrange data</h5>
-  </div>
-  <gb-data-table></gb-data-table>
-  <hr>
-
-  <!---------------------- create new export ---------------------->
-  <div *ngIf="!isLoadingExportDataTypes && exportDataTypes.length > 0">
-    <div class="gb-export-section-banner">
-      <h5>Create new export for the currently selected data</h5>
-    </div>
-
-    <div class="gb-export-format-container">
+  <mat-accordion [multi]="true">
+    <!---------------------- data table ---------------------->
+    <mat-expansion-panel expanded="true">
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          <div class="gb-export-section-banner">
+            <h5>View and arrange data</h5>
```

Comment:
hmm, data table is not really used for arranging the data for surveyTable data view or external export, maybe this text should be changed for data views other than `dataTable`

Reply:
you mean this text `<h5>View and arrange data</h5>` ?

Reply:
yes, for data view other then `dataTable` it could be just `View data`